### ISSUE TITLE
Remove semantic <section>s from footer columns

### ISSUE THREAD

**Issue by cjg89** [UCF/Online-Child-Theme#112]
**Description**
`<section>` elements in our primary footer template part cause non-distinguishable landmark warnings to be thrown due to a lack of a label on each section.

**Why it's Important**
All non-unique landmarks defined in the theme should have labels assigned to them.  Because we can't define a specific enough label for these footer columns, we should remove the landmark association with them.

**Alternatives**
n/a

**Possible Implementation**
Replace `<section>`s with `<div>`s

**Additional context**
Similar: https://github.com/UCF/UCF-WordPress-Theme/issues/76
